src/gtest/test_libzendoo.cpp

```diff
@@ -1973,4 +1973,27 @@ TEST(CctpLibrary, TestInvalidProofVkWhenOversized)
     EXPECT_FALSE(vkInvalid.IsValid());
 
     //TODO: Might be useful to test the same behaviour with bit vector
+}
+
+TEST(CctpLibrary, TestCustomFieldsValidation)
+{
+    for (uint8_t i = 1; i < CHAR_BIT; i++)
```

Comment:
Here we have all possible cases for up-to 1 byte size custom field.
Let's add also some cases with bigger config size in a separate test. I mean, assuming that field element is created  from int\long\full field element. So actually the cases which failed before the fix.

Reply:
Increased unit tests coverage with commit [7267da211e5e0fad639ca9cea7d20a0c6658daf8](https://github.com/HorizenOfficial/zen/commit/7267da211e5e0fad639ca9cea7d20a0c6658daf8).